lib/cocoapods/sources_manager.rb

```diff
@@ -1,4 +1,5 @@
 require 'cocoapods-core/source'
+require 'cocoapods-core/cdn_source'
```

Comment:
why does the sources manager need to require this?

Reply:
True... I added by accident, will remove.